L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.cc

```diff
@@ -0,0 +1,47 @@
+#include "L1Trigger/ME0Trigger/plugins/ME0TriggerProducer.h"
+#include "L1Trigger/ME0Trigger/src/ME0TriggerBuilder.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DataFormats/GEMDigi/interface/ME0TriggerDigiCollection.h"
+#include "Geometry/Records/interface/MuonGeometryRecord.h"
+#include "Geometry/GEMGeometry/interface/ME0Geometry.h"
+
+ME0TriggerProducer::ME0TriggerProducer(const edm::ParameterSet& conf) 
+{
+  me0PadDigiProducer_ = conf.getParameter<edm::InputTag>("ME0PadDigiProducer");
+
+  trigBuilder_.reset( new ME0TriggerBuilder(conf) ); // pass on the conf
```

Comment:
@kpedro88 so initially I copied & modified CSCTriggerPrimitiveDigis, which is "edm::one::EDProducer<edm::one::SharedResources>" because it shares the CSCTriggerGeometry object. There is no ME0TriggerGeometry (I also don't think we need it). So I was wondering if I can change the ME0TriggerPorducer to "edm::stream::EDProducer<>". That way it would be similar to e.g. GEMPadDigiProducer. Although I must admit I'm not that familiar with the  different types of edm producers